bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -786,6 +785,9 @@ public void run() {
                         LOG.error("I/O exception running periodic check", ioe);
                     } catch (ReplicationException.UnavailableException ue) {
                         LOG.error("Underreplication manager unavailable running periodic check", ue);
+                    } finally {
+                        LOG.info("Completed checkAllLedgers in {} milliSeconds", checkAllLedgersDuration);
+                        checkAllLedgersTime.registerSuccessfulEvent(checkAllLedgersDuration, TimeUnit.MILLISECONDS);
```

Comment:
we need to differentiate the if operation completed successfully or not

if an exception occurs we have to call `registerFailedEvent`

Reply:
I have fixed! Please  review again,thanks! @nicoloboschi

Reply:
@nicoloboschi PTAL,thanks!